README.md

```diff
@@ -1,50 +1,70 @@
+    - [Demo iOS App](#demo-ios-app)
+    - [Debug Mode](#debug-mode)
+    - [Test Mode](#test-mode)
+    - [Data Types](#data-types)
+  - [Configure and Initialize the Ironclad Clickwrap SDK](#configure-and-initialize-the-ironclad-clickwrap-sdk)
+  - [PSClickWrapView](#psclickwrapview)
+    - [Preloading Clickwrap Data](#preloading-clickwrap-data)
+    - [Loading Your Clickwrap](#loading-your-clickwrap)
+      - [Interface Builder](#interface-builder)
+        - [IBOutlet your Clickwrap](#iboutlet-your-clickwrap)
+        - [Loads Contracts Into Your Clickwrap](#loads-contracts-into-your-clickwrap)
+      - [Programmatically](#programmatically)
+      - [Configure Contracts Link Tap Behavior](#configure-contracts-link-tap-behavior)
+        - [Import Safari Services](#import-safari-services)
+        - [Configure PSClickWrapView UITextView Delegate](#configure-psclickwrapview-uitextview-delegate)
+        - [Implement UITextViewDegate Protocol and Method](#implement-uitextviewdegate-protocol-and-method)
+      - [Check if Checkbox is Selected](#check-if-checkbox-is-selected)
+      - [Sending Acceptance](#sending-acceptance)
+      - [PSClickWrapViewDelegate](#psclickwrapviewdelegate)
+  - [Checking Acceptance](#checking-acceptance)
+    - [Using the signedStatus Method](#using-the-signedstatus-method)
+    - [Using the PSAcceptanceViewController](#using-the-psacceptanceviewcontroller)
+      - [What it Looks Like](#what-it-looks-like)
+      - [Setting It Up](#setting-it-up)
+      - [PSAcceptanceViewControllerDelegate](#psacceptanceviewcontrollerdelegate)
+        - [Receive Notice of Acceptance](#receive-notice-of-acceptance)
+    - [Using signedStatus Method and Present Alert](#using-signedstatus-method-and-present-alert)
+  - [Sending Activity Manually](#sending-activity-manually)
+  - [Customizing Acceptance Data](#customizing-acceptance-data)
+    - [Connection Data](#connection-data)
+    - [Custom Data](#custom-data)
+      - [Adding Additional Custom Data](#adding-additional-custom-data)
 
 
 ## Requirements
 
 - Xcode 11 or higher
 - Target iOS 10.0 or higher
 - Swift 5.0+
-- PactSafe Published Contracts in Public Group
-- PactSafe Group Key
-- PactSafe Site Access ID
-- PactSafe API Access
+- Ironclad Clickwrap Published Contracts in Public Group
+- Ironclad Clickwrap Group Key
+- Ironclad Clickwrap Site Access ID
+- Ironclad Clickwrap API Access
 
 ## Installation
 
 ### Swift Package Manager
-You can use the [Swift Package Manager](https://swift.org/package-manager/) to install the PactSafe SDK:
+You can use the [Swift Package Manager](https://swift.org/package-manager/) to install the Ironclad Clickwrap SDK:
 
 https://github.com/pactsafe/pactsafe-ios-sdk.git
 
 ### CocoaPods
-You can use [CocoaPods](http://cocoapods.org/) to install the PactSafe SDK by adding it to your Podfile:
+You can use [CocoaPods](http://cocoapods.org/) to install the Ironclad Clickwrap SDK by adding it to your Podfile:
```

Comment:
note- I left PactSafe as is below on line 74. @timothymorse -- please let me know if that no longer makes sense!